lib/tasks/themes.rake

```diff
@@ -292,3 +292,96 @@ task "themes:qunit_all_official" => :environment do |task, args|
   Rake::Task["themes:qunit"].reenable
   Rake::Task["themes:qunit"].invoke("ids", theme_ids_with_qunit_tests.join("|"))
 end
+
+HORIZON_REMOTE_PATHS = %w[
+  github.com/discourse-org/horizon
+  github.com/discourse-org/nextgen-theme
+  github.com/discourse/horizon
+]
+
+def get_horizon_themes
+  Theme.joins(:remote_theme).where(<<~SQL)
+    remote_themes.remote_url LIKE ANY (ARRAY[#{HORIZON_REMOTE_PATHS.map { |path| "'%#{path}%'" }.join(", ")}])
+  SQL
+end
+
+desc "Remove duplicated Horizon themes"
+task "themes:deduplicate_horizon" => :environment do |task, args|
+  horizon_themes = get_horizon_themes
+
+  abort "🟩 No Horizon themes found" if horizon_themes.empty?
+
+  # v0 will handle only one horizon theme
+  abort("⭕ More than one horizon theme installed") if horizon_themes.count > 1
+
+  manual_horizon_theme = horizon_themes.first
```

Comment:
```suggestion
  remote_horizon_theme = horizon_themes.first
```

I think this is just a bit clearer, after reading this and the spec I was a bit confused because I was thinking "manual" meant they "manually uploaded" horizon like a zip